test/riak_search_schema_tests.erl

```diff
@@ -0,0 +1,100 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+
+-module(riak_search_schema_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-define(DEFAULT_DISABLED_JOB_CLASSES, [
+    {riak_search, query}
+]).
+-define(DEFAULT_ENABLED_JOB_CLASSES, [
+]).
+-define(JOB_CLASS_CONFIG_KEY, "riak_core.job_accept_class").
+
+job_class_defaults_test() ->
+    case riak_core_schema() of
+        {true, RCSchema} when erlang:is_list(RCSchema) ->
+            confirm_enabled_job_classes(RCSchema);
+        Ret ->
+            ?assertEqual({error, enoent}, Ret)
```

Comment:
Similar to the riak_kv PR, would like to see some logging here to make it clear to the user that the test was not actually performed.